spec/dummy/app/frontend/javascripts/koi/Composable/ComposableComponent.jsx

```diff
@@ -45,8 +45,8 @@ export default class ComposableComponent extends React.Component {
   }
 
   onAdvancedFormChange(props) {
-    const composition = [ ...this.props.helpers.composition ];
-    composition[this.props.index].advanced = props.values;
+    const composition = { ...this.props.helpers.composition };
+    composition[this.props.helpers.group][this.props.index].advanced = props.values;
     this.props.helpers.replaceComposition(composition);
```

Comment:
A lot the code now reads `helpers.group` or `this.state.group` which is the current visible group. This means that there is no way to modify anything in a non-visible group. I can't think of any use-cases where a non-visible group would need to be updated (reorder, add item, remove item etc.) but something worth keeping in mind I guess.